_posts/en/newsletters/2024-09-06-newsletter.md

```diff
@@ -0,0 +1,169 @@
+---
+title: 'Bitcoin Optech Newsletter #319'
+permalink: /en/newsletters/2024/09/06/
+name: 2024-09-06-newsletter
+slug: 2024-09-06-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Stratum v2 extension for fee revenue sharing:** Filippo Merli
+  [posted][merli stratumfees] to Delving Bitcoin about an extension to
+  [Stratum v2][topic pooled mining] that will allow tracking the amount
+  of fees included in _shares_ when the shares contain transactions
+  selected by an individual miner.  This can be used to adjust the
+  amount paid to the miner by the pool, with miners selecting higher
+  feerate transactions being paid more.
+
+  Merli links to a [paper][merli paper] he co-authored that examines
+  some of the challenges of paying different miners different amounts
+  based on the transactions they select, and which suggests a scheme
+  that is compatible with a _pay per last N shares_ (PPLNS) pooled
+  mining payout scheme.  His post links to two in-progress
+  implementations of the scheme.
+
+- **OP_CAT research fund:** Victor Kolobov [posted][kolobov cat] to the
+  Bitcoin-Dev mailing list to announce a $1 million fund for research
+  into a proposed soft fork to add an [OP_CAT][topic op_cat] opcode.
+  "Topics of interest include but are not limited to: the security
+  implications of OP_CAT activation on Bitcoin, `OP_CAT`-based computing
+  and locking script logic on Bitcoin, applications and protocols
+  utilizing `OP_CAT` on Bitcoin, and general research related to
+  `OP_CAT` and its impact." Submissions must be received by 1 January
+  2025.
+
+- **Mitigating merkle tree vulnerabilities:** Eric Voskuil [posted][voskuil
+  spv] to the Delving Bitcoin discussion thread about the [consensus
+  cleanup soft fork proposal][topic consensus cleanup] (see [Newsletter
+  #296][news296 cleanup]) a request for an update given recent
+  [discussion][voskuil spv dev] on the Bitcoin-Dev mailing list.  In
+  particular, he saw "no justification for the proposed invalidation of
+  64 byte transactions" based on his argument that there's no
+  performance improvement to full nodes in protecting against [merkle
+  tree vulnerabilities][topic merkle tree vulnerabilities] like
+  CVE-2012-2459 by forbidding 64-byte transactions in comparison to
+  other checks that can be performed without a consensus change (and,
+  indeed, those checks are being performed now).  The consensus cleanup
+  proposal champion, Antoine Poinsot, appeared to [agree][poinsot cache]
+  about this full node aspect: "The advantage I initially mentioned
+  about how making 64-bytes transactions invalid could help caching
```

Comment:
Inside a quote so maybe not, but clearly just a typo (or maybe add "[sic]" but I don't love that unless it's formal writing such as an academic paper):
```suggestion
  about how making 64-bytes transactions invalid could help catching
```

Reply:
Leaving as-is.  I believe "caching" is the correct term.  He's talking about storing hashes of consensus invalid blocks in a cache.